backend/routes/passportLogin.js

```diff
@@ -27,14 +27,20 @@ passport.use(new Strategy({
   callbackURL: 'http://localhost:8080/auth/github/callback'
 }, function(accesstoken, refreshToken, profile, done) {
     User.findOne({ gihubId: profile.id }, function(err, user) {
-      console.log(profile);
       
       if (err) return done(err);
       
       if (user) {
+        console.log('done')
         return done(err, user);
       } else {
-        user = new User({ githubId: profile.id })
+        user = new User({ 
```

Comment:
@bonham000 some do, some don't, so it will be hit or miss. I guess we can include it and have that be one of the things they enter later on if it's not there, since we will need a way for people to contact one another if they "match"